python/lsst/daf/butler/core/schema.py

```diff
@@ -44,6 +45,9 @@ class Schema:
     config : `SchemaConfig` or `str`, optional
         Load configuration. Defaults will be read if ``config`` is not
         a `SchemaConfig`.
+    limited : `bool`
+        If True, ignore tables, views, and associated foreign keys whose
```

Comment:
Sometimes `True` is back ticked, here it is not. Be consistent.

Reply:
It should **always** be back ticked in this context.